apiserver/facades/controller/charmrevisionupdater/updater_test.go

```diff
@@ -190,6 +190,7 @@ func (s *charmVersionSuite) TestJujuMetadataHeaderIsSent(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	expected_header := []string{
 		"environment_uuid=" + model.UUID(),
```

Comment:
Is this header still required?

Reply:
Yes because this is used server side by the metrics gathering stuff.
We will remove it when the server is updated.